.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# Contributing at Slack
+
+![Header Image](contributing_header_slack.png)
+
+## Before Contributing
+
+Before contributing, please read our [Code of Conduct](./CODE_OF_CONDUCT.md). We take it very seriously, and expect that you will as well.
+
+## New Issues
+
+Before opening a new issue, please consider:
+
+  - Reading [the documentation](https://github.com/slackhq/SlackTextViewController/blob/master/README.md) and [the changelog](https://github.com/slackhq/SlackTextViewController/blob/master/README.md) first.
+  - Searching for any related issues and avoid creating duplicated issues.
+  - Adding details, diagnoses, screenshots or any type of useful information in existing issues, even if they are marked as closed. The team will still review it.
+  - Use the examples [provided in this repository](https://github.com/slackhq/SlackTextViewController/tree/master/examples).
+  - Taking the time to think of a solution and **[open a pull request]**(#new-pull-requests) for either improving the documentation, fixing a bug or suggesting a feature.
```

Comment:
These aren't rendering as links. Is this desired or were you looking for `[**open a pull request**](#new-pull-requests)`